src/actions/breakpoints/tests/__snapshots__/syncing.spec.js.snap

```diff
@@ -78,7 +78,7 @@ Object {
 
 exports[`reloading debuggee syncs with changed source and an existing disabled BP 1`] = `
 Immutable.Map {
-  http://localhost:8000/examples/magic.js:3:: Object {
+  "http://localhost:8000/examples/magic.js:3:": Object {
```

Comment:
Refer to the Jest Blog to see why snapshots changed up.

https://facebook.github.io/jest/blog/2017/12/18/jest-22.html